### PR TITLE
Update azure_rm_virtualmachineextesion related document

### PR DESCRIPTION
**Pull request by Fred-sun**
##### SUMMARY

Update azure_rm_virtualmachineextesion related document.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

azure_rm_virtualmachineextension/azure_rm_virtualmachineextension_facts
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Fred-sun** [comment]
ready_for_review

File: lib/ansible/modules/cloud/azure/azure_rm_virtualmachineextension.py [hunk 1]
```diff
@@ -34,19 +32,18 @@
...
     name:
         description:
-            - Name of the vm extension
+            - Name of the vm extension.
         required: true
     state:
         description:
-            - Assert the state of the vm extension. Use C(present) to create or update a vm extension and
-              C(absent) to delete a vm extension.
+            - Assert the state of the vm extension. Use C(present) to create or update a vm extension and C(absent) to delete a vm extension.
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_virtualmachineextension.py]:
```suggestion
            - State of the vm extension. Use C(present) to create or update a vm extension and C(absent) to delete a vm extension.
```

File: lib/ansible/modules/cloud/azure/azure_rm_virtualmachineextension.py [hunk 2]
```diff
@@ -102,13 +99,18 @@
 
 RETURN = '''
 state:
-    description: Current state of the vm extension
+    description:
+        - Current state of the vm extension.
     returned: always
     type: dict
+    sample: { "status":"Deleted" }
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_virtualmachineextension.py]: Shouldn't this be `state`, not `status`?
```suggestion
    sample: { "state":"Deleted" }
```

**Review by acozine — COMMENTED** [review]
Looks good, thanks @Fred-sun. Just a couple of minor changes.

**Pull request merged**